EngineAPI/src/main/java/nullengine/block/event/BlockClickEvent.java

```diff
@@ -0,0 +1,30 @@
+package nullengine.block.event;
+
+import nullengine.entity.Entity;
+import nullengine.block.Block;
+import nullengine.event.Event;
+import nullengine.math.BlockPos;
+
+public class BlockClickEvent implements Event {
+    private BlockPos blockPos;
+    private Block block;
+    private Entity entity;
+
```

Comment:
Please do comment like this INLINE!

Like: there should be world reference in this event.

Reply:
Your all commits have many troubles. I don't know what need comment inline!

Reply:
Then what trouble? You should write it down here

Reply:
BlockClickEvent need include `World` field.